src/main/java/org/folio/circulation/domain/CheckInProcessRecords.java

```diff
@@ -218,4 +243,8 @@ public DateTime getCheckInProcessedDateTime() {
   public ItemStatus getItemStatusBeforeCheckIn() {
     return itemStatusBeforeCheckIn;
   }
+
+  public boolean isLostItemFeesRefundedOrCancelled() {
```

Comment:
I think this reads more like natural language using `are` rather than `is`. What do you think?

```suggestion
  public boolean areLostItemFeesRefundedOrCancelled() {
```

Reply:
Sure, agree.